src/app/services/species.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
```

Comment:
I already have a warehouse service in my other PR, once merged, please use that!